### PR TITLE
Added documentation for RefreshMain

### PR DESCRIPTION
**Pull request by chroniclekevinpowe**

### CODE REVIEW

File: docs/antora/modules/tailing/pages/tailing.adoc [hunk 1]
```diff
@@ -360,3 +360,13 @@ Lastly, there is a script for running the reader named `queue_reader.sh` which a
 $ ./bin/queue_reader.sh <options>
 ----
 
+=== Refreshing The Directory Listing For A Queue Using `RefreshMain`
+
+`RefreshMain` can be used to refresh the directory listing for a queue, contained within the tablestore file for the queue. Assuming a project pom file that includes the Chronicle-Queue artifact, you can refresh the directory listing for a queue with the following command:
+
+[source, shell script]
+---
+mvn exec:java -Dexec.mainClass="net.openhft.chronicle.queue.main.RefreshMain" -Dexec.args="myqueue"
```
Comment (JerryShea) [inline docs/antora/modules/tailing/pages/tailing.adoc]: there is also a shell script

**Review by JerryShea — APPROVED** [review]
Nice one! Minor comment

File: docs/antora/modules/tailing/pages/tailing.adoc (hunk 1 above)
Comment (chroniclekevinpowe) [reply docs/antora/modules/tailing/pages/tailing.adoc]: Is it possible that got left on a branch? I don't see it in the bin directory.

Comment (JerryShea) [reply docs/antora/modules/tailing/pages/tailing.adoc]: sorry you are correct! Would you mind adding :) ?

Comment (chroniclekevinpowe) [reply docs/antora/modules/tailing/pages/tailing.adoc]: Not at all - I'll also add documentation for QueueUnlockMain as well, and create the top-level page for command line tools.

File: bin/refresh_queue.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+if [[ "$?" != "0" ]]; then
+    echo "Could not find uber-jar, please run 'mvn package' in the project root"
+    exit 1
+fi
+
+java -cp "$UBER_JAR" net.openhft.chronicle.queue.RefreshMain "$@"
+
+# if running this in CQ source dir, beware of the default system.properties which is for unit testing,
+# and enables resource tracing. This will lead to this tool dying after a while with OOME
```
Comment (JerryShea) [inline bin/refresh_queue.sh]: not true for this tool

**Review by JerryShea — APPROVED** [review]

File: docs/antora/modules/command-line/pages/command_line_tools.adoc [hunk 3]
```diff
@@ -0,0 +1,376 @@
...
+| <<_chroniclehistoryreadermain,*ChronicleHistoryReaderMain*>>
+| Reads history from queue and outputs latency histograms
+
+| <<_chroniclereadermain,*ChronicleReaderMain*>>
+| Displays records from a queue in text form, supporting more complex operations
+
+| <<_chroniclewritermain,*ChronicleWriterMain*>>
+| Write contents of files provided to queue
+
+| <<_dumpmain,*DumpMain*>>
```
Comment (JerryShea) [inline docs/antora/modules/command-line/pages/command_line_tools.adoc:27]: I would put this last as it is a super low level debugging tool

File: docs/antora/modules/command-line/pages/command_line_tools.adoc [hunk 4]
```diff
@@ -0,0 +1,376 @@
...
+----
+$ mvn exec:java -Dexec.mainClass="net.openhft.chronicle.queue.main.DumpMain" -Dexec.args="myqueue"
+----
+
+In the above command invoking *DumpMain*,  _myqueue_ is the directory containing your .cq4 files.
+
+You can also set up any dependent files manually. This requires the `chronicle-queue.jar`, from any version 4.5.3 or later, and that all dependent files are present on the class path. The dependent jars are listed below:
+
+[source, shell script]
+----
```
Comment (tgd) [inline docs/antora/modules/command-line/pages/command_line_tools.adoc]: I recommend removing this as the versions are out of date so might be confusing

File: docs/antora/modules/command-line/pages/command_line_tools.adoc (hunk 3 above)
Comment (chroniclekevinpowe) [reply docs/antora/modules/command-line/pages/command_line_tools.adoc:27]: I hear you - I think there's a case though for keeping alphabetical order for consistency. I'll add a brief note at the start of the entry pointing folks to ChronicleReaderMain

File: docs/antora/modules/command-line/pages/command_line_tools.adoc (hunk 4 above)
Comment (chroniclekevinpowe) [reply docs/antora/modules/command-line/pages/command_line_tools.adoc]: Done - thank you for the feedback!

File: docs/antora/modules/command-line/pages/command_line_tools.adoc [hunk 5]
```diff
@@ -0,0 +1,311 @@
...
+==== Map Enterprise Support
+
+If the queue being archived is backing an Map Enterprise Map (Enterprise Map for ease of reading) `ArchiveRollFiles` can provide additional utility by writing a full snapshot of the map to the current roll cycle before archiving older roll cycle files, ensuring the Enterprise Map contents are safely preserved.
+
+When archiving a queue backing an Enterprise Map, the following parameters must be provided:
+
+|===
+| Parameter | Detail | Required?
+
+| -qbmn | Name of the `QueueBackedMap` | Y
```
Comment (JerryShea) [inline docs/antora/modules/command-line/pages/command_line_tools.adoc]: Map

File: docs/antora/modules/command-line/pages/command_line_tools.adoc [hunk 6]
```diff
@@ -0,0 +1,311 @@
...
+
+Invoking `ArchiveRollFiles` to delete all roll files older than the age specified for `queue1`.
+
+[source, shell script]
+----
+$ mvn exec:java -Dexec.mainClass="software.chronicle.enterprise.queue.util.ArchiveRollFilesMain"
+-Dexec.args="-qc config/queue-config.yaml -qn queue1 -m 2024-11-01T12:15:00 -aa DELETE"
+----
+
+Invoking `ArchiveRollFiles` to archive roll files older than the age specified for a `QueueBackedMap` queue.
```
Comment (JerryShea) [inline docs/antora/modules/command-line/pages/command_line_tools.adoc]: Enterprise Map

**Review by JerryShea — APPROVED** [review]
looks great. Approved with minor comments

File: bin/refresh_queue.sh (hunk 2 above)
Comment (chroniclekevinpowe) [reply bin/refresh_queue.sh]: Now removed

File: docs/antora/modules/command-line/pages/command_line_tools.adoc (hunk 5 above)
Comment (chroniclekevinpowe) [reply docs/antora/modules/command-line/pages/command_line_tools.adoc]: Fixed!

File: docs/antora/modules/command-line/pages/command_line_tools.adoc (hunk 6 above)
Comment (chroniclekevinpowe) [reply docs/antora/modules/command-line/pages/command_line_tools.adoc]: Fixed - thank you!

**Pull request merged**